src/model/HeatExchangerAirToAirSensibleAndLatent.cpp

```diff
@@ -744,5 +804,196 @@ namespace model {
+    if (boost::optional<TableLookup> sensibleEffectivenessofHeatingAirFlowCurve_ =
+          sensibleEffectivenessofHeatingAirFlowCurve()->optionalCast<TableLookup>()) {
+      return sensibleEffectivenessofHeatingAirFlowCurve_->outputValues()[0];
+    } else {
+      return getImpl<detail::HeatExchangerAirToAirSensibleAndLatent_Impl>()->sensibleEffectivenessat100HeatingAirFlow();
+    }
+  }
+
+  double HeatExchangerAirToAirSensibleAndLatent::latentEffectivenessat75HeatingAirFlow() const {
+    DEPRECATED_AT_MSG(3, 8, 0, "As of EnergyPlus 24.1.0, this property ...");
+    if (boost::optional<TableLookup> latentEffectivenessofHeatingAirFlowCurve_ =
+          latentEffectivenessofHeatingAirFlowCurve()->optionalCast<TableLookup>()) {
+      return latentEffectivenessofHeatingAirFlowCurve_->outputValues()[0];
+    } else {
+      return getImpl<detail::HeatExchangerAirToAirSensibleAndLatent_Impl>()->latentEffectivenessat100HeatingAirFlow();
+    }
+  }
+
+  double HeatExchangerAirToAirSensibleAndLatent::sensibleEffectivenessat75CoolingAirFlow() const {
+    DEPRECATED_AT_MSG(3, 8, 0, "As of EnergyPlus 24.1.0, this property ...");
+    if (boost::optional<TableLookup> sensibleEffectivenessofCoolingAirFlowCurve_ =
+          sensibleEffectivenessofCoolingAirFlowCurve()->optionalCast<TableLookup>()) {
+      return sensibleEffectivenessofCoolingAirFlowCurve_->outputValues()[0];
+    } else {
+      return getImpl<detail::HeatExchangerAirToAirSensibleAndLatent_Impl>()->sensibleEffectivenessat100CoolingAirFlow();
+    }
+  }
+
+  double HeatExchangerAirToAirSensibleAndLatent::latentEffectivenessat75CoolingAirFlow() const {
+    DEPRECATED_AT_MSG(3, 8, 0, "As of EnergyPlus 24.1.0, this property ...");
+    if (boost::optional<TableLookup> latentEffectivenessofCoolingAirFlowCurve_ =
+          latentEffectivenessofCoolingAirFlowCurve()->optionalCast<TableLookup>()) {
+      return latentEffectivenessofCoolingAirFlowCurve_->outputValues()[0];
+    } else {
+      return getImpl<detail::HeatExchangerAirToAirSensibleAndLatent_Impl>()->latentEffectivenessat100CoolingAirFlow();
+    }
+  }
+
+  bool HeatExchangerAirToAirSensibleAndLatent::setSensibleEffectivenessat75HeatingAirFlow(double sensibleEffectivenessat75HeatingAirFlow) {
+    DEPRECATED_AT_MSG(3, 8, 0, "As of EnergyPlus 24.1.0, this property ...");
+    if (boost::optional<TableLookup> sensibleEffectivenessofHeatingAirFlowCurve_ =
+          sensibleEffectivenessofHeatingAirFlowCurve()->optionalCast<TableLookup>()) {
+      double sensibleEffectivenessat100HeatingAirFlow =
+        getImpl<detail::HeatExchangerAirToAirSensibleAndLatent_Impl>()->sensibleEffectivenessat100HeatingAirFlow();
+      sensibleEffectivenessofHeatingAirFlowCurve_->setOutputValues(
+        {sensibleEffectivenessat75HeatingAirFlow, sensibleEffectivenessat100HeatingAirFlow});
+      return setSensibleEffectivenessofHeatingAirFlowCurve(sensibleEffectivenessofHeatingAirFlowCurve_.get());
```

Comment:
same problems: can't dereference the curve object if not initialized, and the setting of the output values isn't good since you're not also modifying the independent variable.

Reply:
Side note, but related to a potential final conclusion on whether we should try to maintain backward compatibilty: The `TableLookup` interface is crap from a user experience perspective. The former `TableMultiVariableLookup` was better in that sense since you could do `addPoint(x, y)` and it'd detect if the x value was already there, and you could call `boost::optional<double> yValue(const std::vector<double>& xValues)`

I think we should re-add that capability...